documentation/nginx_config.md

```diff
@@ -0,0 +1,39 @@
+# config
+
+[Back to resource list](../README.md#resources)
+
+## Actions
+
+- `:create`
+- `:delete`
+
+## Properties
+
+| Name                   | Type          | Default                          | Description                                                         | Allowed Values      |
```

Comment:
Is there any reason why we need the `Allowed Values` column if we're not using it?